celi_framework/main.py

```diff
@@ -146,6 +227,20 @@ def instantiate_with_argparse_args(args: argparse.Namespace, cls: Type):
 
 
 if __name__ == "__main__":
+    tool_config_path = get_default_config_path()
+    print(f"Attempting to use config at: {tool_config_path}")
+    config = get_config()
+    if config:
+        run_celi(config)
+    else:
+        logger.error("Failed to load configuration - Check .env file and/or arguments supplied manually. Exiting...")
```

Comment:
If you use exceptions you don't need all this.  If you do keep this code, it needs an exit.  It says it is going to exit, but then it doesn't actually exit.